libvmaf/src/feature/cambi.c

```diff
@@ -720,8 +759,10 @@ static FORCE_INLINE inline double weight_scores_per_scale(double *scores_per_sca
 }
 
 static int cambi_score(VmafPicture *pics, uint32_t *mask_dp, uint16_t window_size, double topk,
```

Comment:
Fine as is, but maybe you want to just take `s` as a parameter here.

Reply:
Yes - @nilfm did something on those lines here: https://github.com/Netflix/vmaf/commit/50d64f97d612fd7f1f01459b2ba3668d717b8e2c
Can be added then.